gradoop-flink/src/test/java/org/gradoop/flink/algorithms/gelly/clusteringcoefficient/GellyVertexClusteringCoefficientDirectedTest.java

```diff
@@ -79,23 +96,18 @@ public void testSpecific() throws Exception {
           v.getPropertyValue(ClusteringCoefficientBase.PROPERTY_KEY_LOCAL).getDouble(), 0.0);
       }
       if (v.getPropertyValue("id").getInt() == 3) {
-        assertEquals("vertex with id 3 has wrong local value, should be 0 or NaN",
-          Double.NaN,
+        assertEquals("vertex with id 3 has wrong local value, should be 0d", 0.0,
```

Comment:
Please have a look few lines above of id 1 and 2 ... the String `should be 1` hast to be `should be 0.5` or? If yes, fix it please because it is related.

Reply:
You're right...I will change it.